src/NHibernate/nhibernate-configuration.xsd

```diff
@@ -92,7 +92,16 @@
 										<xs:enumeration value="cache.region_prefix" />
 										<xs:enumeration value="cache.use_minimal_puts" />
 										<xs:enumeration value="cache.default_expiration" />
-										<xs:enumeration value="query.substitutions" />
+										<xs:enumeration value="cache.use_sliding_expiration">
+									<xs:annotation>
```

Comment:
The indentation here is not correct, due to master being more deeply nested (`xs:union` added on master).